scripts/common/rook.sh

```diff
@@ -216,7 +216,10 @@ function maybe_cleanup_rook() {
 
         if [ "$DID_MIGRATE_ROOK_PVCS" == "1" ] && [ "$DID_MIGRATE_ROOK_OBJECT_STORE" == "1" ]; then
             report_addon_start "rook-ceph-removal" "v1"
-            remove_rook_ceph
+            if ! remove_rook_ceph; then
+                logFail "Unable to remove Rook."
+                return
+            fi
             report_addon_success "rook-ceph-removal" "v1"
```

Comment:
it does not seem like we should report successful here but there does not see any way to report a failure. @laverya what do you recommend?

Reply:
That is a good point.

Reply:
I swear we had a report_addon_failure command

Reply:
congratulations, you can be the first user
https://github.com/replicatedhq/kURL/blob/4d146f4e2ac810f92376100041e2c6388e016203/scripts/common/reporting.sh#L144-L166
just make sure to get rid of the 'unused' comment 😆

Reply:
actually that's probably not quite what you want
maybe make a copy of the https://github.com/replicatedhq/kURL/blob/4d146f4e2ac810f92376100041e2c6388e016203/scripts/common/reporting.sh#L104 function but with the fail_addon endpoint instead